src/booking/booking.py

```diff
@@ -44,7 +44,23 @@
 @click.option('-id', '--user-id', default='', help="user's account.")
 @click.option('-use-opt', '--use-options', default=False, is_flag=True, help="use extra options.")
 @click.option('-ls', '--list-schedule', default=False, is_flag=True, help="list schedule that already booking.")
-def cli(user_id: str = None, use_options: bool = False, list_schedule: bool = False) -> bool:
+@click.option('-rst', '--restart_container', default=False, is_flag=True, help="restart the container.")
+def cli(user_id: str = None, use_options: bool = False, list_schedule: bool = False, restart_container: bool = False) -> bool:
+    
+    if restart_container:
```

Comment:
The priority of checking `restart_container` can be later than checking `user_id`. Therefore, it can only operate `__check_user_passward()` once.